index.html

```diff
@@ -1899,65 +1897,68 @@ <h3>Status</h3>
         <h3>Presentations</h3>
 
         <p>
-<a>Presentations</a> MAY be used to combine and present <a>credentials</a>.
-They can be packaged in such a way that the authorship of the data is
-<a>verifiable</a>. The data in a <a>presentation</a> is often all about the same
-<a>subject</a>, but there is no limit to the number of <a>subjects</a> or
-<a>issuers</a> in the data. The aggregation of information from multiple
-<a>verifiable credentials</a> is a typical use of
-<a>verifiable presentations</a>.
+<a>Verifiable presentations</a> MAY be used to aggregate information from
+multiple <a>verifiable credentials</a>.
         </p>
         <p>
-<a>Verifiable presentations</a> SHOULD be extremely short-lived, and
-bound to a challenge provided by a <a>verifier</a>. Details for accomplishing
-this depend on the securing mechanism, the transport protocol, and
-<a>verifier</a> policies. Unless additional requirements are defined by the
-particular securing mechanism or embedding protocol, a <a>verifier</a> cannot
-generally assume that the <a>verifiable presentation</a> has any correlation
-with the presented <a>verifiable credentials</a>.
+<a>Verifiable presentations</a> SHOULD be extremely short-lived, and bound to a
+challenge provided by a <a>verifier</a>. Details for accomplishing this depend
+on the securing mechanism, the transport protocol, and <a>verifier</a> policies.
+Unless additional requirements are defined by the particular securing mechanism
+or embedding protocol, a <a>verifier</a> cannot generally assume that the
+<a>verifiable presentation</a> has any correlation with the presented
+<a>verifiable credentials</a>.
         </p>
         <p>
-A <a>verifiable presentation</a> is typically composed of the following
-properties:
+The following properties are defined for a <a>verifiable presentation</a>:
         </p>
 
         <dl>
           <dt><var>id</var></dt>
           <dd>
-The <code>id</code> <a>property</a> is optional and MAY be used to provide a
-unique identifier for the <a>presentation</a>. For details related to the use of
-this property, see Section <a href="#identifiers"></a>.
+The <code>id</code> <a>property</a> is optional. It MAY be used to provide a
+unique identifier for the <a>verifiable presentation</a>. If present, the
+normative guidance in Section <a href="#identifiers"></a> MUST be followed.
           </dd>
           <dt><var>type</var></dt>
           <dd>
-The <code>type</code> <a>property</a> is required and expresses the
-type of <a>presentation</a>, such as <code>VerifiablePresentation</code>. For
-details related to the use of this property, see Section <a href="#types"></a>.
+The <code>type</code> <a>property</a> MUST be present. It is used to express the
+type of <a>verifiable presentation</a>. One value of this property MUST be
+<code>VerifiablePresentation</code>, but additional types MAY be included. The
+related normative guidance in Section <a href="#types"></a> MUST be followed.
```

Comment:
Just noting that we don't need the normative statement on types, as all types must follow that guidance anyway. This normative statement doesn't make the implementation do anything above and beyond what's stated in the #types section. I think we can safely remove this statement (in a future PR).